[esx]/es_extended/client/main.lua

```diff
@@ -1,6 +1,19 @@
 local pickups = {}
 local PlayerBank, PlayerMoney = 0,0
 CreateThread(function()
+	--load unloaded ipl's
+  	OnEnterMp()
+  	SetInstancePriorityMode(true)
+  	RequestIpl("FIBlobbyfake")
+  	RequestIpl("DT1_03_Gr_Closed")
+  	RequestIpl("v_tunnel_hole")
+  	RequestIpl("TrevorsMP")
+  	RequestIpl("TrevorsTrailer")
+  	RequestIpl("farm")
+  	RequestIpl("farmint")
+  	RequestIpl("farmint_cap")
+  	RequestIpl("farm_props")
+  	RequestIpl("CS1_02_cf_offmission")
```

Comment:
why????
Literally no need to turn ESX into a IPL loader that u cant even disable